torax/tests/sim_test.py

```diff
@@ -287,6 +286,21 @@ def test_run_simulation(
         ref_name=ref_name,
     )
 
+  @parameterized.parameters([
+      dict(config_name='test_imas.py'),
+      dict(config_name='test_iterhybrid_predictor_corrector_imas.py'),
+  ])
+  def test_imas(self, config_name):
+    """Integration test comparing to reference output from TORAX."""
+    if importlib.util.find_spec('imaspy') is None:
+      self.skipTest('IMASPy optional dependency')
```

Comment:
similar comments to above, we can remove the `skipif` and simply add this to the existing list of sim tests.